### DISCUSSION TITLE
Add subfolder in media folder

### DISCUSSION THREAD

**Discussion by Polm90** [joomengine/Joomla-Component-Builder#780] [Q&A] [answered]
Hi,
I was trying to add a custom file in a subfolder in the media folder.
I have specified as path "media/custom_folder" but when I install the component, the "custom_folder" is not created (and the custom file is not copied). I noticed that in the component xml file, in the media folder list, the "custom_folder" is not specified (however, the file is correctly included in the zip package).

Anyway, if I specify as path "media/images/custom_folder" it works :)

**Comment by Llewellynvdm** [comment, marked as answer]
The best way is not to move the file but the folder.

So in the custom folder, create a folder `jcb_admin_root/custom/custom_folder` and place your file in this folder, and then add the folder to JCB and the target `/media/custom_folder`

This should work, let me know.

**Comment by hennysmafter** [comment]
On the latest Joomla 3 series with the latest staging build from JCB I am encountering the exact same issue.

I created the custom folder. And then in component settings I selected the folder and used as path `/media/custom_folder` and after install it won't show up. 

When changing the path to `/media/images/custom_folder` it does show up.

I am sorry if I should have created a new topic for this. I thought it was better to stay in the conversation.

**Reply by Llewellynvdm** [reply]
Interesting, I have not seen this issue as yet. So I am wondering if it's something to do with your system, but I can't say for sure.

**Reply by hennysmafter** [reply]
Anyway I can help you with diagnosing this? Like providing FTP / SuperUser login? Just let me know, I am more than happy to help out.

**Reply by Llewellynvdm** [reply]
Hi @hennysmafter it will be helpful if we can have live session somehow and helping me see what your doing. We have a upcoming JCB JUG, it could be that we can after do some tests if you like. My days are very full as I am sure yours are too. So lets see if we can workout some time.

To [vote for the JUG time](https://doodle.com/poll/gcvb6sdfpd4znzvy), and to stay in the [loop as to the selected date and time](https://github.com/vdm-io/Joomla-Component-Builder/discussions/826).

**Reply by hennysmafter** [reply]
Dear @Llewellynvdm

I sadly will not be attending the JUG and that is not due to having no interest, but I simply am already swamped with work. There is just not enough time in the day, as you can see with my slow replies, it is just not feasible at this point to attend. 

I would love to help you and myself out via a live session, but if you can plan it in anywhere from the 13th to the 19th, then I will push things out of the way in my calendar to make it happen.

One thing though, I don't mind doing a voice call or discord or whatever, I just don't do video calls so, please don't expect that. I also don't mind sharing my phone number for Whatsapp or Telegram. But I do that in a PM ;-) if you want.

I wish you and all attendees for the JUG a great time.

I am going to end this message with expressing the following. 

Joomla Component Builder is by far the most impressive extension I have ever seen in Joomla's history since Joomla 1. Thank you very much for all your commitment, and also thanks to every single contributor for their contributions.
